tools/cardano-sl-tools.cabal

```diff
@@ -261,8 +243,7 @@ executable cardano-swagger
 
   if flag(for-installer)
     buildable: False
-
-  if !flag(with-wallet) || !flag(with-web)
+  else
     buildable: False
```

Comment:
Ah, I misread it. Then `cardano-swagger` should be moved to `cardano-sl-wallet`. Maybe in another PR. But the point that at least one CI should check that `cardano-swagger` compiles.